test_e2e

```diff
@@ -10,6 +11,16 @@ E2E_DIRECTORY = f"{WORK_DIRECTORY}/e2e"
 #                End to end tests               #
 #################################################
 
-os.environ["PYTHONPATH"] = E2E_DIRECTORY
-os.chdir(E2E_DIRECTORY)
-subprocess.run(["pytest", ".", "-vv"])
+def main(test_filter: str = None):
+    os.environ["PYTHONPATH"] = E2E_DIRECTORY
+    os.chdir(E2E_DIRECTORY)
+    command = ["pytest", ".", "-vv"]
```

Comment:
This can maybe go as const
`E2E_COMMAND = ["pytest", ".", "-vv"]`

Reply:
I thought about it and left it here because it is easier to notice it when debugging, and also at some point in time we would like to make this path "." modifiable.

Reply:
Okay 👍🏻